src/styles/blocks/card.scss

```diff
@@ -0,0 +1,64 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background-color: $background-color-special-white;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  text-align: center;
+  padding-block: 32px 16px;
+
+  transition: all 300ms;
+
+  &:hover {
+    transform: scale(1.2);
```

Comment:
The `transform: scale(1.2)` on hover might affect the layout by causing the card to overlap with other elements. Ensure there is enough space around the card to accommodate this transformation.